corehq/apps/cloudcare/api.py

```diff
@@ -183,8 +184,13 @@ def keys():
                 for bool in status_to_closed_flags(self.status):
                     yield [self.domain, owner_id, bool]
 
-        view_results = CommCareCase.view('hqcase/by_owner', keys=keys,
-                                         include_docs=False, reduce=False)
+        view_results = CommCareCase.view(
+            'hqcase/by_owner',
+            keys=keys,
+            include_docs=False,
+            reduce=False,
+            stale=settings.COUCH_STALE_QUERY
+        )
```

Comment:
that's true, i don't know what's better. stalling for a couple minutes or responding with inconsistent data